packages/fxa-payments-server/server/lib/routes/post-metrics.js

```diff
@@ -34,6 +34,13 @@ const BODY_SCHEMA = {
     .optional(),
 };
 
+function addTime(metricsEvent) {
+  // round the date to the nearest hour.
+  const today = new Date();
+  today.setMinutes(0, 0, 0);
+  metricsEvent.time = today.toISOString();
```

Comment:
This needs to be changed to unix epoch in ms to match the Amplitude API:

> The timestamp of the event in milliseconds since epoch. It will be set to the upload time by default. IE 1396381378123

Reply:
Whoa, that's a good catch. Any idea why doesn't the content-server have to do the same?

Looking here: https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/server/lib/metrics-collector-stderr.js#L20

Reply:
That line is for client metrics.  (Activity metrics and perf metrics also use ISO strings.)  Amplitude event.time is from: https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/server/lib/flow-event.js#L89